### PR TITLE
appended user certificate migration to default certificate generation progress

### PR DESCRIPTION
**Pull request by iszmais**
https://mantis.ilias.de/view.php?id=26745

### CODE REVIEW

**Comment by mjansenDatabay** [comment]
@iszmais Is this still WIP? Maybe we should also involve @qdittrich here, because we wrote most parts of the user certificate migration.

**Comment by iszmais** [comment]
Yes, im still testing it.

**Comment by iszmais** [comment]
@mjansenDatabay Its ready to merge now. I would recommend to refactor code around it since this refurbishing reduced dependencies to now unnecessary code.

File: Services/Certificate/classes/Placeholder/Values/class.ilDefaultPlaceholderValues.php [hunk 1]
```diff
@@ -116,7 +116,6 @@ public function __construct(
 			'DATETIME'           => '',
 			'DATE_COMPLETED'     => '',
 			'DATETIME_COMPLETED' => '',
-			'CLIENT_WEB_DIR'     => ''
```
Comment (mjansenDatabay) [inline Services/Certificate/classes/Placeholder/Values/class.ilDefaultPlaceholderValues.php]: @iszmais This change (only this line) should be also applied for `trunk`

Comment (iszmais) [reply Services/Certificate/classes/Placeholder/Values/class.ilDefaultPlaceholderValues.php]: moved this to another branch to get a better overview.

**Pull request merged**